### PR TITLE
fix(color-picker-flyout): color resets on scroll outside

### PR DESCRIPTION
**Pull request by SamCreasey**
Clickup Ticket: https://app.clickup.com/t/22wejxe

### CODE REVIEW

File: src/components/ColorInputFlyout/ColorPickerFlyout.tsx [hunk 1]
```diff
@@ -27,14 +27,26 @@ export const ColorPickerFlyout: FC<ColorPickerFlyoutProps> = ({
...
+    const handleOpenChange = (isOpen: boolean) => {
+        if (!isOpen) {
+            onClose?.();
+        }
+        setOpen(isOpen);
+    };
+
+    const handleClick = () => {
+        setOpen(false);
+        onClick && onClick();
```
Comment (triggertoo) [inline src/components/ColorInputFlyout/ColorPickerFlyout.tsx:39]:
```suggestion
        onClick?.();
```
according to `onClose`-call

File: src/components/Flyout/Flyout.tsx [hunk 2]
```diff
@@ -209,14 +209,7 @@ export const Flyout: FC<FlyoutProps> = ({
...
                             fixedHeader={fixedHeader}
-                            onClick={
-                                onClick
-                                    ? () => {
-                                          onClick();
-                                          close();
-                                      }
-                                    : undefined
-                            }
+                            onClick={onClick ? onClick : undefined}
```
Comment (triggertoo) [inline src/components/Flyout/Flyout.tsx:212]: will the dialogue still work/close correctly in the other components using it?

Comment (SamCreasey) [reply src/components/Flyout/Flyout.tsx:212]: Yes as far as I could see :)

File: src/components/ColorInputFlyout/ColorPickerFlyout.tsx (hunk 1 above)
Comment (SamCreasey) [reply src/components/ColorInputFlyout/ColorPickerFlyout.tsx:39]: Fixed

**Review by triggertoo — APPROVED** [review]

**Pull request merged**